content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/enforce-2fa.md

```diff
@@ -7,25 +7,26 @@ description: >
 ---
 
 {{% pageinfo %}}
-As an [Organization Owner](/getting-started/glossary/#organization-owner), you can enforce two-factor-authentication (2FA) for all users within your organization.
+As an [Organization Owner](/getting-started/glossary/#organization-owner), you can enforce two-factor authentication (2FA) for all organization users who authenticate with their email and password.
 {{% /pageinfo %}}
 
-2FA enforcement affects organization users and pentest collaborators, including the following roles:
+## Important to Know
 
-{{% owner-member-team-member %}}
+Note the following about 2FA enforcement:
 
-{{< alert title="Note" color="primary" >}}
-If your organization uses [SAML-based single sign-on (SSO)](/getting-started/sign-in/#saml-sso), you bypass any 2FA enforcement. Your SAML provider may already require 2FA.
-{{< /alert >}}
+- 2FA enforcement only applies to **users who authenticate with their email and password**.
+- 2FA enforcement doesn't apply to single sign-on (SSO). This includes [SAML single sign-on (SSO)](/getting-started/sign-in/#saml-sso) and Google authentication.
+- If your organization has SAML SSO enforced, you can't enforce 2FA. Your SAML provider may already require 2FA.
+- 2FA enforcement affects users with the following roles:
+    {{% owner-member-team-member %}}
 
 ## Enforce 2FA
 
-1. Navigate to **Settings** > **Identity & Access**.
-1. Under **Two-Factor Authentication**, select **Enforce 2FA**.<br><br>
-  ![Enforce 2FA for your organization](/deepdive/Enforce2FA.png "Enforce 2FA for your organization")
-1. In the overlay that appears, select **Yes, enforce 2FA** to confirm.
+1. Navigate to **Settings** > **Security**.
+1. Under **Enforce Two-Factor Authentication (2FA)**, turn on the toggle.
+1. In the overlay that appears, confirm your action.
 
-Users who haven't yet configured 2FA on their account get an email notification. They need to enable 2FA before signing in. For details, see steps 3–5 described in [Enable Two-Factor Authentication](/platform-deep-dive/cobalt-account/account-settings/#enable-two-factor-authentication).
+Users who haven't yet configured 2FA on their account get an email notification. They must enable 2FA upon their next sign-in.
 
 As an Organization Owner who enforces 2FA, you see an overlay prompting you to [enable 2FA](/platform-deep-dive/cobalt-account/account-settings/#enable-two-factor-authentication)—if you haven't already.
 
```

Comment:
Is this true for the new implementation? Should we say "enforced" instead of "configured"?

You see a yellow warning icon for a user when:
- The user hasn't enabled 2FA—regardless of the 2FA enforcement; and
- Your organization doesn't have SAML SSO configured.

Reply:
" As an Organization Owner who enforces 2FA, you see an overlay prompting you to [enable 2FA](/platform-deep-dive/cobalt-account/account-settings/#enable-two-factor-authentication)—if you haven't already." No longer true, we will just force them to setup next login. 
![Uploading Screenshot 2023-04-13 at 11.47.29 AM.png…]()

Yellow icon still appears on people page

Reply:
Removed this sentence:

" As an Organization Owner who enforces 2FA, you see an overlay prompting you to [enable 2FA](https://github.com/platform-deep-dive/cobalt-account/account-settings/#enable-two-factor-authentication)—if you haven't already."